src/RequestHandlerSwooleRunner.php

```diff
@@ -193,24 +206,101 @@ function (Throwable $exception) use ($serverRequestErrorResponseGenerator) : Res
             ));
         }
 
-        $this->logger = $logger ?: new StdoutLogger();
+        $this->logger = $logger ? : new StdoutLogger();
+        $this->pidManager = $pidManager;
     }
 
     /**
      * Run the application
      */
     public function run() : void
     {
-        $this->swooleHttpServer->on('start', [$this, 'onStart']);
-        $this->swooleHttpServer->on('request', [$this, 'onRequest']);
-        $this->swooleHttpServer->start();
+        $opts = new Getopt([
+            'daemonize|d'  => 'daemonize the swoole server process',
+            'worker_num|n=i' => 'set worker nums',
```

Comment:
- `s/worker_num/num_workers/` (makes it align with the short flag better, too; when it changes, you'll need to change the way you retrieve it, too)
- `s/set worker nums/set number of workers/`.
- Indicate the default, if no number is provided.